### ISSUE TITLE
Step Line Chart

### ISSUE THREAD

**Issue by XD-DENG** [XD-DENG/ECharts2Shiny#24]
``` javascript
option = {
    title: {
        text: 'Step Line'
    },
    tooltip: {
        trigger: 'axis'
    },
    legend: {
        data:['Step Start', 'Step Middle', 'Step End']
    },
    grid: {
        left: '3%',
        right: '4%',
        bottom: '3%',
        containLabel: true
    },
    toolbox: {
        feature: {
            saveAsImage: {}
        }
    },
    xAxis: {
        type: 'category',
        data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
    },
    yAxis: {
        type: 'value'
    },
    series: [
        {
            name:'Step Start',
            type:'line',
            step: 'start',
            symbol:'circle',
            symbolSize:30,
            data:[120, 132, 101, 134, 90, 230, 210]
        },
        {
            name:'Step Middle',
            type:'line',
            step: 'middle',
            symbol:'triangle',
            symbolSize:20,
            data:[220, 282, 201, 234, 290, 430, 410]
        },
        {
            name:'Step End',
            type:'line',
            step: 'end',
            symbol: 'rect',
            symbolSize:20,
            data:[450, 432, 401, 454, 590, 530, 510]
        }
    ]
};

```

**Comment by XD-DENG** [comment]
<img width="896" alt="screen shot 2016-07-23 at 11 02 46 am" src="https://cloud.githubusercontent.com/assets/11539188/17075414/0a5bfd80-50c5-11e6-9c16-467ae710d2d3.png">

**Comment by XD-DENG** [comment]
addressed with commit 4533ae2bfefd058edfa1fb4a555e52f0aeb57d0e
